frontend/src/components/school/SchoolContacts.vue

```diff
@@ -152,16 +157,17 @@ export default {
     getThisSchoolsContacts(){
       this.loadingCount += 1;
       let searchSchoolID = this.schoolID ? this.schoolID: this.userInfo.activeInstituteIdentifier;
-
       ApiService.apiAxios.get(`${ApiRoutes.school.SCHOOL_DETAILS_BY_ID}/` + searchSchoolID)
         .then(response => {
           this.schoolContacts = new Map();
           response.data.contacts.forEach(contact => {
-            if (!this.schoolContacts.has(contact.schoolContactTypeCode)) {
-              this.schoolContacts.set(contact.schoolContactTypeCode, [contact]);
-              return;
+            if(!isExpired(contact)){
+              if (!this.schoolContacts.has(contact.schoolContactTypeCode)) {
+                this.schoolContacts.set(contact.schoolContactTypeCode, [contact]);
+                return;
```

Comment:
Please remove the `return` in favor of using an else block

Reply:
Ok done.